examples/sanitizers/BUILD

```diff
@@ -3,14 +3,18 @@ load(
     "xcodeproj",
 )
 load(
-    "//:xcodeproj_targets.bzl",
+    ":xcodeproj_targets.bzl",
+    "SCHEME_AUTOGENERATION_MODE",
     "XCODEPROJ_TARGETS",
+    "get_xcode_schemes",
 )
 
 xcodeproj(
     name = "xcodeproj",
     build_mode = "bazel",
     project_name = "Sanitizers",
+    scheme_autogeneration_mode = SCHEME_AUTOGENERATION_MODE,
+    schemes = get_xcode_schemes(),
```

Comment:
You'll need to set this in `test/fixtures/BUILD` as well.

Reply:
`examples` is present in the root `.bazelignore` and is being filtered out. What would be your suggestions to go about it?

Reply:
I mean you need to edit this: https://github.com/buildbuddy-io/rules_xcodeproj/blob/d316ac374b4f38d73d882647e0b36593cfe2fb32/examples/sanitizers/test/fixtures/BUILD#L10

Reply:
Oh okay, makes sense! Updated.